### PR TITLE
[ES] added content/es/service-discovery

### LINKED ISSUE #1788: [Spanish] localize 'Service Discovery'
Localize the `content/en/service-discovery.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `content/es/service-discovery`.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1788 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/service-discovery.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Descubrimiento de Servicios
```
Comment (ramrodo) [inline content/es/service-discovery.md:2]:
```suggestion
title: Descubrimiento de Servicio
```
En la traducción original dice `Service Discovery` en singular. Podemos manejarlo igual en todas las referencias de la localización.

File: content/es/service-discovery.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Problema que aborda
+
+Las arquitecturas cloud native son dinámicas y fluidas, es decir que están en constante cambio.
+Una aplicación [contenedorizada](/es/containerization/) se ejecutará y detendrá varias veces en su tiempo de vida.
+Cada vez que eso sucede, obtendrá una nueva dirección y
+cualquier otra aplicación que quiera encontrarla necesita una herramienta que provea su nueva dirección.
+
+## ¿Cómo ayuda?
+
+El descubrimiento de servicios mantiene una lista de aplicaciones en una red para que puedan encontrarse unas a otras cuando sea necesario.
```
Comment (ramrodo) [inline content/es/service-discovery.md:22]:
```suggestion
El descubrimiento de servicio mantiene una lista de aplicaciones en una red para que puedan encontrarse unas a otras cuando sea necesario.
```

Mismo comentario que el anterior

File: content/es/service-discovery.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Provee un espacio común donde encontrar y potencialmente identificar servicios individuales.
+Los motores de descubrimiento de servicios son herramientas similares a las bases de datos que almacenan información referente a qué servicios existen y cómo localizarlos.
```
Comment (ramrodo) [inline content/es/service-discovery.md:24]:
```suggestion
Los motores de descubrimiento de servicio son herramientas similares a las bases de datos que almacenan información referente a qué servicios existen y cómo localizarlos.
```

Mismo comentario que el anterior

**Review by ramrodo — CHANGES_REQUESTED** [review]
Algunas observaciones sobre el singular de la palabra **Servicios**

**Review by 92nqb — APPROVED** [review]
LGTM

**Review by ramrodo — APPROVED** [review]
Thanks! LGTM

**Pull request merged**